### ISSUE TITLE
Failed to build helloworld with nim 2.0 on Mac M2

### ISSUE THREAD

**Issue by laoshaw** [yglukhov/nimx#523]
/Users/tester/.nimble/pkgs2/clipboard-0.1.0-beaedd7095bcaefeb82d0de6b01d9134a8686e27/clipboard/clipboard_mac.nim(136, 6) Error: cannot bind another '=destroy' to: MacClipboard:ObjectType; previous declaration was constructed here implicitly: /Users/tester/.nimble/pkgs2/clipboard-0.1.0-beaedd7095bcaefeb82d0de6b01d9134a8686e27/clipboard/clipboard_mac.nim(68, 24)

**Comment by miutamihai** [comment]
Getting the same on an M1

**Comment by diegovskytl** [comment]
@laoshaw  @miutamihai are you trying with nim ARM or x86?

**Comment by diegovskytl** [comment]
> see the title, M2 is ARM, thanks.

yes, but you can use nim x86 through Rosetta in Apple Silicon. Therefore my question.
